lib/container_server.go

```diff
@@ -53,7 +54,41 @@ func (c *ContainerServer) Runtime() *oci.Runtime {
 
 // Hooks returns the oci hooks for the ContainerServer
 func (c *ContainerServer) Hooks() map[string]HookParams {
-	return c.hooks
+	hooks := map[string]HookParams{}
+	c.hooksLock.Lock()
+	for key, h := range c.hooks {
+		hooks[key] = h
+	}
+	c.hooksLock.Unlock()
+	return hooks
+}
+
+// RemoveHook removes an hook by name
+func (c *ContainerServer) RemoveHook(hook string) {
+	c.hooksLock.Lock()
+	defer c.hooksLock.Unlock()
+	_, ok := c.hooks[hook]
```

Comment:
Nit: Do this in a single line
if _, ok := c.hooks[hook]; ok {
...

Reply:
Yup, changing